examples/step-3/doc/results.dox

```diff
@@ -138,6 +138,63 @@ suggestions:
   discontinuous boundary values, zero on three sides of the square, and one on
   the fourth.
 
+  <li>
+  Use triangles: As mentioned in the results section of step-1, for
+  historical reasons, almost all tutorial programs for deal.II are
+  written using quadrilateral or hexahedral meshes. But deal.II also
+  supports triangular and tetrahedral meshes. So a good experiment would
+  be to replace the mesh used here by a triangular mesh.
+
+  This is *almost* trivial. First, as discussed in step-1, we may want
+  to start with the quadrilateral mesh we are already creating, and
+  then convert it into a triangular one. You can do that by replacing
+  the first line of `Step3::make_grid()` by the following code:
+  @code
+  Triangulation<2> triangulation_quad;
+  GridGenerator::hyper_cube(triangulation_quad);
+  GridGenerator::convert_hypercube_to_simplex_mesh (triangulation_quad,
+                                                    triangulation);
```

Comment:
I think you might want to mention that the this conversion function creates 8 triangles out of a single quadrilateral via https://github.com/dealii/dealii/blob/77ac9098c18b35709e63c6948f0da7d6b0b51258/source/grid/grid_generator.cc#L7932-L7943 so to reach the same number of global degrees of freedom, one needs to reduce the number of global refinements by 1. But I believe one needs more refinement to reach the same accuracy with simplices, so maybe 5 is not that wrong after all.

Reply:
Yes, good idea. I mention it in passing, but don't dwell on it. That's for a later program.